catalogue/webapp/components/ExpandedImage/ExpandedImage.js

```diff
@@ -187,7 +189,7 @@ const ExpandedImage = ({
     function closeOnEscape(event: KeyboardEvent) {
       if (event.key !== 'Escape') return;
 
-      setExpandedImageId('');
+      setExpandedImage(undefined);
```

Comment:
`undefined` over `null`?

Reply:
`undefined` because that will be the state value before any image is selected from `useState()` so I don't want to introduce 2 "empty" types. In general I am not a fan of `null` for reasons which lots of very boring people have written about elsewhere on the internet.

Reply:
I think across the codebase we have settled for `null` over `undefined` as it makes it explicit that that is what we are doing by having `setState(null)`. I will bet we haven't done this universally though.